mu-plugins/blocks/table-of-contents/index.php

```diff
@@ -36,7 +36,9 @@ function init() {
  * @return string Returns the block markup.
  */
 function render( $attributes, $content, $block ) {
-	$items = get_headings( get_the_content() );
+	$post = get_post();
```

Comment:
I should have used the `postId` from the context in this block, oops. If you want to, you could make that change too (see https://github.com/WordPress/wporg-developer/pull/181/commits/105875f5d3880791ac01594daa6c236b976b5fae). Not a blocker though, that doesn't change the work you've got here.

Reply:
Done in https://github.com/WordPress/wporg-mu-plugins/pull/332/commits/ca79e8c4ffcdf80828246e3a99bff975765518f9. 👍